docs/chapters/architecture/Lifecycle/index.rst

```diff
@@ -0,0 +1,170 @@
+:orphan:
+
+.. _Life cycle-Management:
+
+Life cycle management
+=====================
+
+Introduction and goals
+----------------------
+
+The life cycle cluster is a set of components that work closely with each other
+to define and implement the functionality required in the life cycle of a typical
+IVI scope.
+
+Requirements Overview
+^^^^^^^^^^^^^^^^^^^^^
+
+- Life cycle Management must override blocking consumers on shutdown
+- Supply Management must be able to route hardware power events
+- The system should detect the wake up reason to prevent or modify start-up if
+  functionality is not needed
+- The system functionality to store necessary data is depending on the shutdown
+  mode
+- In diagnostic or flash sessions the life cycle is controlled by diagnostic
+- Before a managed reset/reboot, the system provides a degraded mode in order
+  to maintain main user services
+- The system has the ability to delay the shutdown depending on application state
+- The system has the ability to manage a timeout to shutdown depending on application
+  state
+- The system has to follow the global car operation modes
+- There must be a normal operation mode
+- The system can be woken up by the user pressing the Power On button on the ECU
+- The system can be woken up by the user inserting a CD/DVD in the drive
+- The system can be woken up by the user pressing the CD/DVD eject button
+- The system can be woken up by an active bus signal
+- The system can be shutdown by a specific Clamp State notification
+- The system can be shutdown by the ECU after a product defined timeout when there is
+  no active Clamp State
+- The system can be instructed to shutdown by an external tester
+- There must be a parking mode
+- There must be an operation mode for transporting the cars
+- There must be an operation mode for building the cars in factories
+- There must be an operation mode for flashing devices
+- There must be a thermal management
+- Following power up, the system is gradually functional
+- The life cycle must provide watchdog functionality for system applications
+- The ECU must handle an immediate power off
+- The life cycle must ensure safety critical applications can always be started by the
+  driver
+- To provide Log&Trace data for problem analysis the DLT must be supported
+- Accordingly to its state, the system could generate a reset/reboot
+- The system must support a real time clock and an uptime clock
+- For specific states, the system is not allowed to reboot
+- There must be a communication mode
+- There must be a shutdown delay mode
+- The Life cycle Management can stop applications or reboot the system if a CPU load
+  threshold is reached
+- The emergency call must be possible in all situations where the customer is present
+- The system can be woken up from network/bus signals
+- Life cycle Management must be able to perform a fast system shutdown when requested [1]_
```

Comment:
This is not the way we should add references. See [here](https://pelux.io/software-factory/master/swf-blueprint/docs/articles/intro/how-to-use-the-swf-platform-blueprint.html)

Reply:
hm... that looks identical to me

Reply:
Below is the raw view of the last part of the document:

`An example, and reference, SWF Deployment is the PELUX Baseline SWF
Deployment [#pelux-swf]_.`

`.. [#pelux-swf] http://pelux.io/software-factory`

The `#pelux-swf` will take the role of reference.